### ISSUE TITLE
Usage is more confusing then useful

### ISSUE THREAD

**Issue by srcspider** [sindresorhus/gulp-imagemin#111]
Your `Usage` section implies we have to add each plugin to use it. 

The various options are also not intuitive, since you have `progressive: true` there, but that doesn't really apply to pngquant, but is actually meant to apply to optipng. 

There are some hints there but its way too hard tell that the example is suppose to mean: "set these options to default plugins and add this pngquant plugin" 
It's far more likely for it to get interpreted as: "use ONLY pngquant to optimize, here are the options for it" 

```
var gulp = require('gulp');
var imagemin = require('gulp-imagemin');
var pngquant = require('imagemin-pngquant');

gulp.task('default', function () {
    return gulp.src('src/images/*')
        .pipe(imagemin({
            progressive: true,
            svgoPlugins: [{removeViewBox: false}],
            use: [pngquant()]
        }))
        .pipe(gulp.dest('dist/images'));
});
```

You mention "Comes bundled with the following optimizers" in API but that can mean anything really. It didn't even occur to me by bundled you mean all of them are activated by default before looking at the code.

Might be better to mention that all those plugins are enabled even with out doing anything and that you can use the `use` section to add other plugins into `imagemin`, such as https://www.npmjs.com/package/imagemin-webp

I think the most confusing thing about it there is that you use `pngquat` which is very easy to confuse with `optipng`, because when you read the api list and the above code you only really parse "thing that optimizes png" really (you might even see the difference in name but still think its the same thing; because why would anyone mention 2 png optimizers?).

Another really confusing bit there is what exactly happens when I have 3 png optimizers? Does it pass though all of them? Does it needless compile with all of them then discard 2 of them at random?

**Comment by noderat** [comment]
I agree, the usage is incredibly confusing and I can't seem to tell if pngquant is even working!

**Comment by RoberMac** [comment]
I have to say I can't understand the docs.

**Comment by sindresorhus** [comment]
Fixed in 3.0.0. This module ships with good defaults still, for the common case, but it's easily overridden.
